pymc_marketing/model_builder.py

```diff
@@ -777,6 +777,43 @@ def sample_posterior_predictive(
 
         return az.extract(post_pred, variable_name, combined=combined)
 
+    def compute_log_likelihood(
```

Comment:
It seems this is a wrapper of `pm.compute_log_likelihood` and, therefore, redundant. Shall we remove this and simply use `pm.compute_log_likelihood` when needed